src/Dapr.Actors/DaprHttpInteractor.cs

```diff
@@ -461,5 +459,26 @@ private HttpClient CreateHttpClient()
 
             return httpClientInstance;
         }
+
+        private void AddDaprApiTokenHeader(HttpRequestMessage request)
+        {
+            if (!string.IsNullOrWhiteSpace(this.daprApiToken))
+            {
+                request.Headers.Add("dapr-api-token", this.daprApiToken);
+                return;
+            }
+
+            // Check if the env variable DAPR_API_TOKEN is set and use it as the default value
+            var apiToken = Environment.GetEnvironmentVariable("DAPR_API_TOKEN");
```

Comment:
We should be able to isolate this code to the options class. Putting it here makes this class have more dependencies taht are hard to test.

Reply:
Whereas, it's easy to rationalize a get/set property without unit tests 😆